go_router/test/go_router_test.dart

```diff
@@ -30,72 +41,54 @@ void main() {
       final router = _router(routes);
       router.go('/');
       final matches = router.routerDelegate.matches;
-      expect(matches.length, 1);
-      expect(matches[0].fullpath, '/');
-      expect(router.screenFor(matches[0]).runtimeType, ErrorScreen);
+      expect(matches, hasLength(1));
+      expect(matches.first.fullpath, '/');
+      expect(router.screenFor(matches.first).runtimeType, ErrorScreen);
     });
 
     test('empty path', () {
-      try {
+      expect(() {
         GoRoute(path: '');
-        expect(false, true);
-      } on Exception catch (ex) {
-        dump(ex);
-      }
+      }, throwsException);
```

Comment:
this has the same effect on the test but throws away useful info for when I was building the text (am I seeing the exception I expected to see). this is fine now; just was giving you the history.

Reply:
👍

Reply:
There doesn't seem to be a good way to debug log these exceptions when using `throwsException`... It might be worth investigating.